solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -1,84 +1,102 @@
+
+Execute `cluster-setup.sh` to setup K8s cluster on your VM.  
+If you want to run pods on to the master node as well, give first input parameter for `cluster-setup.sh` script as true else false.  
+If you have single node cluster, pass the input parameter as true.
+
 ```
-      ./cluster-setup.sh true
+./cluster-setup.sh true
 ```
 
-## Deploy Cortx Stack on K8s Cluster
+## Deploy the Cortx Stack on K8s Cluster
+
+Execute `cortx-deploy.sh` to deploy Cortx Cluster on your K8s Cluster.
 
-   - Execute `cortx-deploy.sh` to deploy Cortx Cluster on your K8s Cluster.
 ```
-      export SOLUTION_CONFIG_TYPE=automated && ./cortx-deploy.sh --cortx-cluster
+export SOLUTION_CONFIG_TYPE=automated && ./cortx-deploy.sh --cortx-cluster
 ```
 
-   Note:
-   1. CORTX_SCRIPTS_BRANCH - If you want to use another cortx-K8s branch then export this variable with your branch (Default value CORTX_SCRIPTS_BRANCH="v0.2.1").
-   2. CORTX_SCRIPTS_REPO - If you want to use another cortx-K8s repo (like your fork), then export this variable with your repo (Default value CORTX_SCRIPTS_REPO="Seagate/cortx-k8s").
-   3. CORTX_ALL_IMAGE - In automated case we are using latest cortx-all image. If you want to use different image then export that image by this variable(Default value CORTX_ALL_IMAGE=ghcr.io/seagate/cortx-all:2.0.0-latest).
-   4. CORTX_SERVER_IMAGE - Also cortx-server image if you want to use different then export this variable with that image (Default value CORTX_SERVER_IMAGE=ghcr.io/seagate/cortx-rgw:2.0.0-latest).
-   5. CORTX_DATA_IMAGE - Also cortx-data image if you want to use different then export this variable with that image (Default value CORTX_DATA_IMAGE=ghcr.io/seagate/cortx-data:2.0.0-latest).
-   6. CONTROL_EXTERNAL_NODEPORT - If you want to use different port for control service, then export this variable with another port (Default value CONTROL_EXTERNAL_NODEPORT="31169").
-   7. S3_EXTERNAL_HTTP_NODEPORT - If you want to use different port for HTTP Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTP_NODEPORT="30080").
-   8. S3_EXTERNAL_HTTPS_NODEPORT - If you want to use different port for HTTPS Port to IO service, then export this variable with another port (Default value S3_EXTERNAL_HTTPS_NODEPORT="30443").
-   9. SOLUTION_CONFIG_TYPE - There are two types config for solution yaml file manual and automated. Automated means solution.yaml is created automatically but it needs VM with required specifications and manual means we need to create solution.yaml file and replace it at script location. If you wanted to create solution yaml manually, then create it and place it at script location and configure SOLUTION_CONFIG_TYPE variable as manual (Default value SOLUTION_CONFIG_TYPE=manual).
+Note:
+   1. CORTX_SCRIPTS_BRANCH : If you want to use another cortx-K8s branch then export this variable with your branch (Default value CORTX_SCRIPTS_BRANCH="v0.2.1").
```

Comment:
Maybe we should mention the default values of all the parameters also. We can use below format,
Variable Name |  Default value | description